spec/controllers/contacts_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe ContactsController, type: :controller do
+  let(:user) { create(:user) }
+
+  let!(:stafford) { create(:contact, name: "stafford") }
+  let!(:winsongreen) { create(:contact, name: "winson green") }
+  let!(:brinsford) { create(:contact, name: "brinsford") }
+
+  describe "GET #index" do
+    before do
+      sign_in user
+    end
+
+    it "returns an alphabetically sorted list of contacts" do
+      # Assuming the controller action assigns @contacts
+      get :index
+
+      # Extract the contacts from the controller's instance variable
+      contacts = assigns(:contacts.name)
```

Comment:
I'm not sure what `.name` is here, or indeed how this is working with it!

Reply:
I've looked into this and it is a new method that can be called on symbols in Ruby 3 - https://www.bigbinary.com/blog/ruby-3-adds-symbol-name - which basically returns the symbol as a string.

It's not needed here anyway.

Reply:
Thank you for the above.  I have removed it from the code base.